execution_engine_testing/test_support/src/wasm_test_builder.rs

```diff
@@ -62,6 +63,7 @@ use casper_types::{
     DeployHash, DeployInfo, EraId, Gas, Key, KeyTag, PublicKey, RuntimeArgs, StoredValue, Transfer,
     TransferAddr, URef, U512,
 };
+use walkdir::WalkDir;
```

Comment:
This one can go to the top where we have other 3rd part crates imported.

Reply:
5f072c245